packages/api/cms-api/src/user-permissions/user.resolver.ts

```diff
@@ -47,4 +39,14 @@ export class UserResolver {
         this.userService.unsetImpersonatedUser(request);
         return true;
     }
+
+    @ResolveField(() => Number)
+    async permissionsCount(@Parent() user: User): Promise<number> {
+        return (await this.userService.getPermissions(user)).length;
+    }
+
+    @ResolveField(() => Number)
```

Comment:
Graphql differentiates between Float and Int. if you change that to `Int` (imported from `@nestjs/graphql`) it would be more correct.